validator/slashing-protection/local/standard-protection-format/export.go

```diff
@@ -120,7 +120,20 @@ func signedAttestationsByPubKey(ctx context.Context, validatorDB db.Database, pu
 		return nil, nil
 	}
 	signedAttestations := make([]*format.SignedAttestation, 0)
-	for _, att := range history {
+	for i := 0; i < len(history); i++ {
+		att := history[i]
+		// Special edge case due to a bug in Prysm's old slashing protection schema. The bug
+		// manifests itself as the first entry in attester slashing protection history
+		// having a target epoch greater than the next entry in the list. If this manifests,
+		// we skip it to protect users. This check is the best trade-off we can make at
+		// the moment without creating any false positive slashable attestation exports.
+		// More information on the bug can found in https://github.com/prysmaticlabs/prysm/issues/8893.
+		if i == 0 && len(history) > 1 {
```

Comment:
lets make the check stronger here, if I understand the bug correctly isn't source also 0 ? That way we don't ignore any valid attestations.

Reply:
yes, good idea

Reply:
do we want to add a test for this as well?